### PR TITLE
aws-lambda-helper does not export a saveToS3() function anymore

### PR DESCRIPTION
**Pull request by hassy**

### CODE REVIEW

**Comment by tomgco** [comment]
Something does not like this PR... I have re-ran and still no luck. (could be the API though)

**Comment by hassy** [comment]
Tests that I think are unrelated to this change are breaking

**Comment by hassy** [comment]
https://github.com/numo-labs/lambda-ne-classic-package-provider/blob/master/test/parse_sns.test.js#L40

Today is the 15th! 👏

File: test/parse_sns.test.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ describe('parse_sns', function () {
 describe('get_age', function () {
   it('gets the age in years of a passenger given her birth date', function (done) {
     var DOB = '1986-07-14'; // Jimmy's test!
-    assert(parse_sns.get_age(DOB) === 29);
+    assert(parse_sns.get_age(DOB) === 30);
```
Comment (tomgco) [inline test/parse_sns.test.js]: Happy birthday jimmy I guess.

**Pull request merged**